### PR TITLE
Fixed MD040 errors in REST and GraphQL topics

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) fixes all MD040 errors in REST and GraphQL topics.

The MD040 rule requires that a language be specified for fenced code blocks. However, it also identifies indented code blocks, so you may see changes in this PR that replace indentation with fences.

This is one of many PRs related to #5252.

### CODE REVIEW

File: guides/v2.3/graphql/develop/debugging.md [hunk 1]
```diff
@@ -14,13 +14,13 @@ You can use Xdebug for debugging the PHP execution of a GraphQL query just as yo
...
 
-```
+```http
 http://<magento2-3-server>/graphql?XDEBUG_SESSION_START=PHPSTORM
 ```
 
 You can also enable an Xdebug connection for a particular request by setting the corresponding header parameter.
 
-```
+```http
```
Comment (keharper) [inline guides/v2.3/graphql/develop/debugging.md]:
```suggestion
```text
```

File: guides/v2.3/graphql/reference/cms.md [hunk 2]
```diff
@@ -55,7 +55,7 @@ The following query returns information about the "404 Not Found" CMS page:
 
 **Request**
 
-```
+```json
```
Comment (keharper) [inline guides/v2.3/graphql/reference/cms.md]:
```suggestion
```graphql
```

File: guides/v2.3/graphql/reference/cms.md [hunk 3]
```diff
@@ -95,7 +95,7 @@ The following query returns information about the `login-data` block:
...
```
Comment (keharper) [inline guides/v2.3/graphql/reference/cms.md]:
```suggestion
```graphql
```

File: guides/v2.3/graphql/reference/customer.md [hunk 4]
```diff
@@ -108,7 +108,7 @@ The following call returns information about the logged-in customer. Provide the
 
 **Request**
 
-``` text
+```text
```
Comment (keharper) [inline guides/v2.3/graphql/reference/customer.md]:
```suggestion
```graphql
```

File: guides/v2.3/graphql/reference/wishlist.md [hunk 5]
```diff
@@ -38,7 +38,7 @@ The following query returns the customer's wish list:
 
 **Request**
 
-```
+```json
```
Comment (keharper) [inline guides/v2.3/graphql/reference/wishlist.md]:
```suggestion
```graphql
```

**Comment by jeff-matthews** [comment]
Thanks @keharper

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**